app/views/devise/confirmations/new.html.erb

```diff
@@ -1,3 +1,17 @@
+<style>
+      .button1 {
```

Comment:
All these copypasted style blocks are crying out for a shared css file 🙏

Reply:
Ok, I think I fixed it by changing the custom.scss file.